packages/contracts/src/contract-data.ts

```diff
@@ -9,25 +9,25 @@ import { Network } from './connect-contracts'
  */
 
 const Mainnet__Lib_AddressManager = require('../deployments/mainnet/Lib_AddressManager.json')
-const Mainnet__OVM_CanonicalTransactionChain = require('../deployments/mainnet/OVM_CanonicalTransactionChain.json')
-const Mainnet__OVM_L1CrossDomainMessenger = require('../deployments/mainnet/OVM_L1CrossDomainMessenger.json')
-const Mainnet__OVM_StateCommitmentChain = require('../deployments/mainnet/OVM_StateCommitmentChain.json')
+const Mainnet__CanonicalTransactionChain = require('../deployments/mainnet/CanonicalTransactionChain.json')
+const Mainnet__OVM_L1CrossDomainMessenger = require('../deployments/mainnet/L1CrossDomainMessenger.json')
```

Comment:
```suggestion
const Mainnet__L1CrossDomainMessenger = require('../deployments/mainnet/L1CrossDomainMessenger.json')
```

And same in a few places below

Reply:
I wasn't sure whether this file needs to remain backward compatible, but I have done the updates now assuming it will be used against the new regenesis deployment.